app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
```

Comment:
The endpoint should also receive a list of model names, for which we should create `Model` objects.